lib/core/src/lib/snackbar-content/snackbar-content.component.scss

```diff
@@ -0,0 +1,16 @@
+:host {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    .snackbar-message {
+        margin: 0;
+    }
+
+    .mat-button {
+        //padding: 0;
```

Comment:
commented out code should not be committed to repository

Reply:
thanks for noticing that @DenysVuika. I were looking for any commented code but did not notice this single line ;)

Reply:
done